extensions/amp-live-list/0.1/live-list-manager.js

```diff
@@ -167,8 +167,8 @@ export class LiveListManager {
   updateLiveList_(liveList) {
     const id = liveList.getAttribute('id');
     user().assert(id, 'amp-live-list must have an id.');
-    user().assert(id in this.liveLists_, `amp-live-list#${id} found but did ` +
-        'not exist on original page load.');
+    user().assert(id in this.liveLists_,
+        'amp-live-list# %s found but did not exist on original page load.', id);
```

Comment:
same comment as above about space and `#`

Reply:
Done